shlex.js

```diff
@@ -125,9 +72,9 @@ class Shlexer {
         if (char === '\n') {
           // An escaped newline just means to continue the command on the next
           // line. We just need to ignore it.
-        } else if (!in_quote || char === in_quote || this.escapes.includes(char)) {
+        } else if (!inQuote || char === inQuote || this.escapes.includes(char)) {
           token = (token || '') + char
-        } else {
+        } else { // Ignores newlines
```

Comment:
I don't think this comment is necessary or clear. The part of the code that handles newlines is above.

Reply:
Yeah this was accidentally left in. I tried changing this statement to `else if (char !== '\n')` with that comment to avoid the empty statement above, but it didn't end up working.